src/main/java/seedu/duke/entities/Order.java

```diff
@@ -32,4 +32,12 @@ public double getTotalPrice() {
         return totalPrice;
     }
 
+    public void printOrder() {
```

Comment:
One small suggestion: Instead of having printOrder() methods in both Order and OrderController classes, I think it might be better to pass the string representation back to the printOrder() method in OrderController... U can discuss with ziyan and see if it is better

Reply:
OK